sample_usage/print_nodes.py

```diff
@@ -21,40 +31,62 @@ def print_node(filename, node_class):
 
     for node in module.nodes_of_class(node_class):
         if node.fromlineno == node.end_lineno:
-            line = source_lines[node.fromlineno - 1]
-            out = [Style.DIM +
-                   line[:node.col_offset] +
-                   Style.BRIGHT + Fore.LIGHTBLACK_EX + Back.BLACK +
-                   line[node.col_offset:node.end_col_offset] +
-                   Style.DIM +
-                   line[node.end_col_offset:] + Style.RESET_ALL]
+            line = source_lines[node.fromlineno - 1]  # string
+            out = [
+                Style.DIM +
+                line[:node.col_offset] +
+
+                # The key part:
+                MAIN_COL +
+                line[node.col_offset : node.end_col_offset] +
+
+                Style.DIM + Fore.RESET + Back.RESET +
+                line[node.end_col_offset:]
+            ]
         else:
-            first_line = source_lines[node.fromlineno - 1]
-            last_line = source_lines[node.end_lineno - 1]
-            middle_lines = source_lines[node.fromlineno: node.end_lineno - 1]
-
-            out = (
-                [Style.DIM +
-                 first_line[:node.col_offset] +
-                 Style.BRIGHT + Fore.LIGHTBLACK_EX + Back.BLACK +
-                 first_line[node.col_offset:]] +
+            first_line = source_lines[node.fromlineno - 1]  # string
+            middle_lines = source_lines[node.fromlineno: node.end_lineno - 1]  # list
+            last_line = source_lines[node.end_lineno - 1]  # string
+
+            if middle_lines:
```

Comment:
Do not convert `middle_lines` into a single string here. It *is* a good idea to call map though, to get the colouring right:

```python
middle_lines = list(map(wrap_color, middle_lines))
```